valhalla/midgard/constants.h

```diff
@@ -36,10 +36,13 @@ constexpr double kDecimeterPerSectoKPH = 0.36; // dm/s to km/h
 
 // Angular measures
 constexpr float kPi = 3.14159265f;
+constexpr double kPiD = M_PI;
```

Comment:
i think this might cause trouble on some systems (i have a vague memory that its not always defined in the same place or something) would you mind just dropping the constant in here?